index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Professional Portfolio GERSON SANCHEZ</title>
+
+</head>
+
+<body>
+
+    <div class="container">
+        <header>
```

Comment:
[OPTIONAL] The margin between your header elements is not similar to the figma design

- [ ] Your design
<img width="679" alt="Screenshot 2022-09-23 at 15 59 52" src="https://user-images.githubusercontent.com/48546992/191966426-3670a8df-8814-4e47-b19a-af2cbe72882e.png">

- [ ] Figma design
<img width="664" alt="Screenshot 2022-09-23 at 15 59 12" src="https://user-images.githubusercontent.com/48546992/191966473-930b02bc-6621-492d-b691-dc2e8425116e.png">


- [ ]  Please note that your color and fonts also do not match the figma design. Please fix this.